src/main/java/org/generationcp/middleware/domain/gms/GermplasmListNewColumnsInfo.java

```diff
@@ -119,4 +122,20 @@ public void print(int indent) {
 		return new HashSet<>();
 	}
 
+	public List<String> getAddedColumnCurrentSort() {
```

Comment:
As discussed, please work on a solution that will work for both cases when added columns have been saved and when re-opened again.